agent/api/eni/eni.go

```diff
@@ -96,6 +107,20 @@ func (eni *ENI) GetSubnetGatewayIPV4Address() string {
 	return eni.SubnetGatewayIPV4Address
 }
 
+// IsRegularENI returns true if the ENI is a standard/regular ENI. That is, if it
+// has its association protocol as standard. To be backwards compatible, if the
+// association protocol is not set for an ENI, it's considered a standard ENI as well.
+func (eni *ENI) IsRegularENI() bool {
```

Comment:
I like the name `IsStandardENI` > `IsRegularENI` .
Since the types are standard and branch.

Reply:
done